routes/users.js

```diff
@@ -11,7 +11,7 @@ const { getUserBadges } = require("../controllers/badges");
 router.post("/verify", authenticate, users.verifyUser);
 router.get("/userId/:userId", users.getUserById);
 router.patch("/self", authenticate, userValidator.updateUser, users.updateSelf);
-router.get("/", authenticate, userValidator.getUsers, users.getUsers);
+router.get("/", userValidator.getUsers, users.getUsers);
```

Comment:
Will we be able to send a body in get request? if yes how? if no why do we need the request data validator?

Reply:
HTTP GET method should not have a request body.